docs/getting-started/quickstart.md

```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
+version: 0
+
+doc_mapping:
+  field_mappings:
+    - name: title
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: body
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: url
+      type: text
+      tokenizer: raw
+      stored: true # Field is stored. 
+
+search_settings:
+  # If you do not specify fields in your query, those fields will be used.
+  default_search_fields: [title, body]
 ```
 
-Now we can create the index with the command:
+Let's make use of the default configuration file provided in quickwit installation throughout this guide.
+by setting the `QW_CONFIG` environment variable.
 
 ```bash
-
-./quickwit new --index-uri file:///your-path-to-your-index/wikipedia --index-config-path ./wikipedia_index_config.json --index wikipedia --metastore-uri file:///path-to-your-metastore
+export QW_CONFIG=./config/quickwit.yaml
 ```
 
-The metastore is like a directory of indexes that has it's own storage and some meta information about the indexes.
-
-To simplify our command, we now assume that the index and metastore are to be created in the current directory and use the `pwd` command to specify the index uri:
+Now we can create the index with the command:
 
 ```bash
-./quickwit new --index-uri file://$(pwd)/wikipedia --index-config-path ./wikipedia_index_config.json --index wikipedia --metastore-uri file://$(pwd)/metastore/
+./quickwit index create --index wikipedia --index-config=./wikipedia_index_config.yaml
 ```
 
-Check that an empty directory `$(pwd)/wikipedia` has been created, Quickwit will write index files here and a `quickwit.json` which contains the [index metadata](../overview/architecture.md#index-metadata).
+Check that a directory `$(pwd)/qwdata/wikipedia` has been created, Quickwit will write index files here and a `quickwit.json` which contains the [index metadata](../overview/architecture.md#index-metadata).
 You're now ready to fill the index.
 
-:::note
-
-The `index id` will be useful when using the search REST API.
-
-:::
-
 
 ## Let's add some documents
 
-Currently Quickwit can index new line delimited json [ndjson](http://ndjson.org/) datasets.
+Quickwit can index data from many [sources](./sources.md). We will use a new line delimited json [ndjson](http://ndjson.org/) datasets as our data source.
 Let's download [a bunch of wikipedia articles (10 000)](https://quickwit-datasets-public.s3.amazonaws.com/wiki-articles-10000.json) in [ndjson](http://ndjson.org/) format and index it.
 
 ```bash
 # Download the first 10_000 Wikipedia articles.
 curl -o wiki-articles-10000.json https://quickwit-datasets-public.s3.amazonaws.com/wiki-articles-10000.json
 
 # Index our 10k documents.
-./quickwit index --index wikipedia --data-dir-path ./wikipedia --metastore-uri file://$(pwd)/metastore --input-path wiki-articles-10000.json
+./quickwit index ingest --index wikipedia --input-path=./wiki-articles-10000.json
```

Comment:
can you make the command consistent everywhere? there are some `arg=value` with `=` here and there